recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,6 @@
-libnetcdf:
-  - 4.8
+c_compiler_version:        # [linux]
+  - 11.2.0                 # [linux]
+cxx_compiler_version:      # [linux]
+  - 11.2.0                 # [linux]
+fortran_compiler_version:  # [linux or osx]
+  - 11.2.0                 # [linux or osx]
```

Comment:
Probably we can remove the local cbc.yaml as the new compilers are already in the global cbc.yaml

Reply:
yes. it reflects now global cbc.  I think that we should remove it here, but add actual openblas key here instead.  It is superflous right now too, but well, it indicates for what version it was built, and it makes things more clear how to process on future update?